services/movie_session.py

```diff
@@ -0,0 +1,58 @@
+from django.core.exceptions import ObjectDoesNotExist
+from db.models import MovieSession, CinemaHall, Movie
+from datetime import date, datetime
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int,
+) -> MovieSession:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id,
+    )
+
+
+def get_movies_sessions(session_date: str = "") -> MovieSession | None:
+    if session_date:
+        year, month, day = session_date.split("-")
+        return MovieSession.objects.filter(

```

Comment:
refactor this to use only one return. 
Also you may pass string as is without conversion - django will take care of everything under the hood